src/sas/sascalc/fit/BumpsFitting.py

```diff
@@ -412,8 +415,14 @@ def abort_test():
     success = best is not None
     try:
         stderr = fitdriver.stderr() if success else None
-        cov = (fitdriver.cov() if not hasattr(fitdriver.fitter, 'state') else
-               np.cov(fitdriver.fitter.state.draw().points.T))
+        if not hasattr(fitdriver.fitter, 'state'):
+            cov = fitdriver.cov()
+        else:
+            x = fitdriver.fitter.state.draw().points
+            if x.shape[1] > 1:
+                cov = np.cov(x.T, bias=1)
```

Comment:
`bias` is a bool, should be `True` or `False` not `1`. Should be `True` in this case, assuming we want the estimate of the population covariance.